airqo-platform/AirQo-frontend
# Mob 185 showcase design changes

#### Summary of Changes (What does this PR do?)

- Adding design changes to the app Tour.
- Corresponding figma designs are [here](https://www.figma.com/file/agN9GOkAbhXfGDEXYCKwBu/Mobile-App?node-id=65%3A11883&t=eUlRRrQ668ny84Io-0).

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Run the app.
- Ensure the showcase is running as expected and the visuals similar to te corresponding [figma designs.](https://www.figma.com/file/agN9GOkAbhXfGDEXYCKwBu/Mobile-App?node-id=65%3A11883&t=eUlRRrQ668ny84Io-0)
- Ensure nothing is broken.

#### What are the relevant tickets?


- [MOB-185](https://airqoteam.atlassian.net/browse/MOB-185) (If exists)

#### Screenshots (optional)


[MOB-185]: https://airqoteam.atlassian.net/browse/MOB-185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
